src/components/FormValidator.js

```diff
@@ -55,6 +54,7 @@ class FormValidator{
           this._submitButton.classList.add(this._inactiveButtonSelector)
           this._submitButton.disabled = true;
       } else {
+        this._submitButton = this._form.querySelector(this._submitButtonSelector);
```

Comment:
`this._submitButton = this._form.querySelector(this._submitButtonSelector);`


Please, delete it


The button has been found in line 65 as ` this._submitButton` already